src/core/simulation.jl

```diff
@@ -1011,6 +1011,7 @@ function _execute!(
     sim::Simulation,
     store;
     cache_size_mib = 1024,
+    min_cache_flush_size_mib = MIN_CACHE_FLUSH_SIZE_MiB,
```

Comment:
We can now customize the flush size by passing this kwarg to `execute!`.

Reply:
Should we also increase the size of the default value or do something smart to set that up?

Reply:
The code will flush this value or the size of each array-at-a-time-point, whichever is bigger. I think this is OK.

Reply:
My question is how does a user know they need to increase the flush size?

Reply:
My hope is that users never need to know about it. There is no need to tempt users to set it. If there ever is a problem then this will allow us to customize behavior without having to change any code.